sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -529,4 +535,37 @@ public ValueHolder(V v) {
             throw new IllegalStateException("Failed to get POJO.", e);
         }
     }
+
+    public static <T> T parse(byte[] item, Class<T> itemClassType) {
+        if (Utils.isEmpty(item)) {
+            return null;
+        }
+        try {
+            return getSimpleObjectMapper().readValue(item, itemClassType);
+        } catch (IOException e) {
+            throw new IllegalStateException("Failed to get POJO.", e);
+        }
+    }
+
+    public static ByteBuffer serializeJsonToByteBuffer(ObjectMapper objectMapper, Object object) {
+        try {
+            ByteBufferOutputStream byteBufferOutputStream = new ByteBufferOutputStream();
+            objectMapper.writeValue(byteBufferOutputStream, object);
```

Comment:
Will it result in re-allocations (dynamic size)?

Reply:
yes